packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -31,11 +32,38 @@ class SmsSendView extends FormView {
   mustAuth = true;
   template = Template;
 
+  initialize() {
+    this.events = assign(this.events, {
+      'input .phone-number': '_validatePhoneNumber',
+    });
+  }
+
+  afterRender() {
+    // Make this a jQuery object so we can use the validation helpers
+    this.phoneField = this.el.querySelector('.phone-number');
+    this.$phoneField = this.$(this.phoneField);
+    this.submitButton = this.el.querySelector('.sms-send');
+
+    this._validatePhoneNumber();
```

Comment:
Tiny nit: can we add a small comment here with a reason why we need to run this on `afterRender`? Or maybe do a check for `if (this.formPrefill.get('phoneNumber')) {` (I think that's right) for one less render + kind of self-documenting code?